src/main/java/org/jcodings/transcode/TranscodingManager.java

```diff
@@ -140,11 +140,11 @@ private static EConv create(byte[] sourceEncodingName, byte[] destinationEncodin
     }
 
     /**
-     * Rubinius and JRuby process Encoding::Converter options flags differently.  Rubinius splits the processing
+     * We and JRuby process Encoding::Converter options flags differently.  We split the processing
      * between initial setup and the replacement value setup, whereas JRuby handles them all during initial setup.
      * We figure out what flags JRuby additionally expects to be set and set them to satisfy EConv.
      */
-    private static int rubiniusToJRubyFlags(int flags) {
+    private static int toJRubyFlags(int flags) {
```

Comment:
Unrelated to your change here, but we could call this `toJCodingsFlags`.

Reply:
Will do.